recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ requirements:
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
     - setuptools
+    - libgcc  # [linux]
```

Comment:
Not sure to what you are referring. For the most part, conda-forge tries to use system compilers and system libraries related to those compilers.